### PR TITLE
leaflet-custom.css: Make it full screen

### PR DESCRIPTION
**Pull request by sbrl**
Here's that cosmetic PR I talked about! Hope you like it :-)

### CODE REVIEW

**Review by est31 — CHANGES_REQUESTED** [review]
Could you remove the sans-serif change? Otherwise looks good

**Comment by sbrl** [comment]
Erm can do! Personally I hate _Times New Roman_ with a passion, so I have it set to be a part of my default css rule template.

File: www/leaflet-custom.css [hunk 1]
```diff
@@ -1,7 +1,12 @@
+html, body { font-size: 100%; }
```
Comment (est31) [inline www/leaflet-custom.css]: What is this doing?

Comment (sbrl) [reply www/leaflet-custom.css]: Ah, oops! That's part of my template too. I'll remove it.

It makes the `rem` css unit behave  as expected if you need to use it.

**Review by est31 — APPROVED** [review]

**Pull request merged**